src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -26,11 +28,22 @@
   "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" befindet sich bereits im Shaderpaket-Ordner!",
   "options.iris.shaders.disabled": "Shader: Deaktiviert",
   "options.iris.shaders.enabled": "Shader: Aktiviert",
+  "options.iris.back": "Zurück",
+  "options.iris.reset": "Zurücksetzen",
+  "options.iris.reset.tooltip": "Setze ALLE Einstellungen zurück und wende diese an?",
+  "options.iris.reset.tooltip.holdShift": "Halte SHIFT und klicke um zurückzusetzen",
+  "options.iris.setToDefault": "Setze Einstellung zurück auf Standardwert?",
```

Comment:
Same goes for this sentence. It just sounds a bit weird when you don't have a proper subject